src/txn/txn_recover.c

```diff
@@ -556,7 +558,8 @@ __wt_txn_recover(WT_SESSION_IMPL *session)
 	 */
 	WT_ERR(session->iface.checkpoint(&session->iface, "force=1"));
 
-done:	FLD_SET(conn->log_flags, WT_CONN_LOG_RECOVER_DONE);
+done:	FLD_SET(conn->log_flags,
+	    WT_CONN_LOG_RECOVER_DIRTY | WT_CONN_LOG_RECOVER_DONE);
```

Comment:
Thanks for the push back!  I didn't read closely enough and thought it was a FLD_CLR.  I have fixed this and added a clear statement.